### PR TITLE
[MRG] New testimonials on features and integration

### PR DESCRIPTION
**Pull request by michelfernandez**
- Added SVG for archive assistance
- Added new feature archive assistance (automatic deletation) in 3 languages DE, FR, EN

### CODE REVIEW

**Comment by michelfernandez** [comment]
Since we currently only offer archive assistance for enterprises, I have removed it here in the features so that we can complete the pull-request with the new testimonials.

File: site/content/english/integration.md [hunk 1]
```diff
@@ -87,14 +88,20 @@ technological security with simplicity.
 
 {{< /features-container >}}
 
+
+{{< content text=center top=4 bottom=4 width=narrow >}}
+{{% markdown %}}
+&nbsp;
```
Comment (fursinger) [inline site/content/english/integration.md:94]: We shouldn't do that? What was the intention behind it?

Comment (michelfernandez) [reply site/content/english/integration.md:94]: Yes, I wanted more space so that the testimonials look more balanced. Couldn't solve it any other way.

Comment (fursinger) [reply site/content/english/integration.md:94]: Okay. Let me have a look at it.

**Comment by fursinger** [comment]
Dominik should have a quick look because I edited the features component.

**Comment by michelfernandez** [comment]
Sounds good!

**Comment by dkumpf1** [comment]
[DEPLOY-PREVIEW]

**Tested: OK**

The new change looks good. I tested this on Chrome/FF/Chromium and mobile on Chrome/native and the changes are good.

**Pull request merged**